src/realm/util/terminate.cpp

```diff
@@ -128,15 +128,17 @@ REALM_NORETURN static void terminate_internal(std::stringstream& ss) noexcept
 {
     util::Backtrace::capture().print(ss);
 
-    ss << "!!! IMPORTANT: Please report this at https://github.com/realm/realm-core/issues/new/choose";
+    ss << "\n!!! IMPORTANT: Please report this at https://github.com/realm/realm-core/issues/new/choose";
+
     if (termination_notification_callback) {
         termination_notification_callback(ss.str().c_str());
     }
     else {
-        std::cerr << ss.rdbuf() << '\n';
+        std::cerr << ss.rdbuf();
         std::string thread_name;
         if (Thread::get_name(thread_name))
-            std::cerr << "Thread name: " << thread_name << "\n";
+            std::cerr << "\nThread name: " << thread_name;
+        std::cerr << std::endl;
```

Comment:
Can we avoid to use `std::endl` it flushes the buffer along doing many other things, we just need `\n`.